src/components/Input/__tests__/DropDownGroup.spec.js

```diff
@@ -31,12 +31,18 @@ describe("DropDownGroup", () => {
     ).toMatchSnapshot();
   });
 
-  it("renders default input correctly when the isOpen prop with a value of true is passed", () => {
+  it("renders input correctly when the isOpen prop with a value of true is passed", () => {
     expect(
       renderComponent({ isOpen: true }).container.firstChild
     ).toMatchSnapshot();
   });
 
+  it("renders input correctly when the keywordSearch prop with a value of false is passed", () => {
+    expect(
+      renderComponent({ keywordSearch: true }).container.firstChild
```

Comment:
Test's description says that keywordSearch of value false is passed, but test itself passes `true`?

Reply:
Whoops :D